### PR TITLE
implement tracing in the account service

### PR DESCRIPTION
**Pull request by C0rby**
Added tracing to the accounts service.

### CODE REVIEW

File: accounts/pkg/service/v0/tracing.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+	span.Annotate([]trace.Attribute{
+		trace.StringAttribute("id", req.Id),
+	}, "Execute Accounts.GetAccount handler")
+
+	return t.next.GetAccount(ctx, req, acc)
+}
+
+func (t tracing) CreateAccount(ctx context.Context, req *v0proto.CreateAccountRequest, acc *v0proto.Account) error {
+	ctx, span := trace.StartSpan(ctx, "Accounts.CreateAccoutn")
```
Comment (kulmann) [inline accounts/pkg/service/v0/tracing.go:47]: Typo in `CreateAccount`

Comment (C0rby) [reply accounts/pkg/service/v0/tracing.go:47]: Nice catch!

**Review by kulmann — CHANGES_REQUESTED** [review]
Please have a look at `ocis/pkg/command/settings.go` and `ocis/pkg/command/accounts.go`. You can see a TODO in the accounts command, that should be fixed in this PR by applying the ocis tracing config to the accounts tracing config, the same way as done in the settings command.

**Review by kulmann — APPROVED** [review]
❤️

**Pull request merged**